Story/AriapetQuests.cs

```diff
@@ -0,0 +1,43 @@
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreStory.cs
+using Skua.Core.Interfaces;
+
+public class AriapetQuests {
+  public IScriptInterface Bot =>IScriptInterface.Instance;
+  public CoreBots Core =>CoreBots.Instance;
+  public CoreStory Story = new();
+
+  public void ScriptMain(IScriptInterface bot) {
+    Core.SetOptions();
+    Storyline();
+    Core.SetOptions(false);
+  }
+
+  public void Storyline() {
+    if (Core.isCompletedBefore(46)) return;
```

Comment:
also if i can remember correctly AriaPet is member only quests? so if it is member pls add above is completed check:
if (!Core.IsMember)
return;
to stop the bot for any one who doesn't have active member shit